RecoJets/JetProducers/python/HEPTopTaggerV2_cfi.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+
+# Jet Clustering Defaults
+from RecoJets.JetProducers.AnomalousCellParameters_cfi import *
+from RecoJets.JetProducers.PFJetParameters_cfi import *
+
+# CHS
+chsForHTT = cms.EDFilter("CandPtrSelector", src = cms.InputTag("packedPFCandidates"), cut = cms.string("fromPV"))  
+
+# Schedule HEPTopTagger
+HEPTopTaggerV2 = cms.EDProducer(
```

Comment:
please use lower case for module instance labels.
hepTopTaggerV2
If this name is acceptable, please also rename the file to be hepTopTaggerV2_cff.py 
- name should match the instance
- cfi files should not define multiple module instances (this one has chsForHTT and hepTopTaggerV2), use cff in this case